server/routerlicious/packages/routerlicious-base/src/test/alfred/api.spec.ts

```diff
@@ -378,10 +385,46 @@ describe("Routerlicious", () => {
 						defaultProducer,
 						defaultDocumentRepository,
 						defaultDocumentDeleteService,
+						null,
+						null,
+						defaultCollaborationSessionEventEmitter,
 					);
 					supertest = request(app);
 				});
 
+				describe("api/v1", () => {
+					it("/api/v1/:tenantId/:id/broadcast-signal 200", async () => {
```

Comment:
And we can name the next test `"Invalid request content"`.
```suggestion
				describe("/api/v1/:tenantId/:id/broadcast-signal", () => {
					it("Successful request", async () => {
```